cmd/delete.go

```diff
@@ -154,6 +93,59 @@ var deleteCmd = &cobra.Command{
 	DisableAutoGenTag: true,
 }
 
+func generateFRComponentDeleteCommands() {
+	type componentProperties struct {
+		artifactName string
+		hidden       bool
+		aliases      []string
+	}
+	var componentList = map[string]componentProperties{
```

Comment:
we autogenerate the commands for the individual manifests based on these specs. Note some of them are hidden to not pollute the docs.